plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -315,6 +301,44 @@ static void ffmpeg_source_start(struct ffmpeg_source *s)
 	}
 }
 
+static void *ffpmeg_source_reconnect(void *data)
+{
+	os_sleep_ms(10000);
```

Comment:
This should be a configurable delay or ideally, reconnect ASAP with exponential backoff.